tests/integration/email.test.js

```diff
@@ -121,6 +123,21 @@ describe('EMAIL ROUTES - POST emails/send-welcome', () => {
         sinon.assert.calledWithExactly(sendinBlueTransporter);
       });
 
+    it('should send a welcoming email as coach and receiver is future registered trainee in auth company',
```

Comment:
que penses tu de mettre "as sender is coach" j'ai eu un peu de mal a comprendre la phrase